content/english/java-tictactoe/activity-1.md

```diff
@@ -1,6 +1,108 @@
 ---
-title: "Create the Class"
-description: ""
+title: "1. Set Up the Board!"
+description: "Using print statements prompt the game rules and board"
 date: 2020-07-23T00:00:00Z
 weight: 2
----
\ No newline at end of file
+---
+
+{{% notice tip %}}
+#### Prerequisites (from Java: Basics) 
+- activity-1: [Print Statements and Comments](../../java-basics/activity-1)
```

Comment:
These links will probably not open in a new tab. I suggest you might use an HTML <a> tag to insert this link, with target="_blank" attribute, so that the student can return to this page after they finished the basics.

Reply:
fixed